save-frontend/build.gradle.kts

```diff
@@ -49,6 +49,9 @@ kotlin {
                     }
                 }
             }
+            commonWebpackConfig {
+                cssSupport.enabled = true
```

Comment:
Why is this required?

Reply:
This is necessary for external CSS style to work. 
It is called with _kotlinext.js.require("example.css")_